app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,33 @@
+class FoodsController < ApplicationController
+  def index
+    @user = current_user
+    @foods = Food.where(user: current_user) || []
+  end
+
+  def new
+    @food = Food.new
+    render 'new'
+  end
+
+  def create
+    @food = Food.new(food_params)
+    @food.user = current_user
```

Comment:
**OPTIONAL** - To help make your code cleaner and more coincise, you could do `current_user.foods.new(food_params`